code/game/machinery/vending/vendor_types/medical.dm

```diff
@@ -17,40 +17,8 @@
 	var/datum/health_scan/last_health_display
 
 	var/healthscan = TRUE
-	var/list/chem_refill = list(
-		/obj/item/reagent_container/hypospray/autoinjector/bicaridine,
-		/obj/item/reagent_container/hypospray/autoinjector/dexalinp,
-		/obj/item/reagent_container/hypospray/autoinjector/adrenaline,,
-		/obj/item/reagent_container/hypospray/autoinjector/inaprovaline,
-		/obj/item/reagent_container/hypospray/autoinjector/kelotane,
-		/obj/item/reagent_container/hypospray/autoinjector/oxycodone,
-		/obj/item/reagent_container/hypospray/autoinjector/tramadol,
-		/obj/item/reagent_container/hypospray/autoinjector/tricord,
-		/obj/item/reagent_container/hypospray/autoinjector/emergency,
-		/obj/item/reagent_container/hypospray/autoinjector/skillless,
-		/obj/item/reagent_container/hypospray/autoinjector/skillless/tramadol,
-
-		/obj/item/reagent_container/hypospray/autoinjector/bicaridine/skillless,
-		/obj/item/reagent_container/hypospray/autoinjector/kelotane/skillless,
-		/obj/item/reagent_container/hypospray/autoinjector/tramadol/skillless,
-		/obj/item/reagent_container/hypospray/autoinjector/tricord/skillless,
-
-		/obj/item/reagent_container/glass/bottle/bicaridine,
-		/obj/item/reagent_container/glass/bottle/antitoxin,
-		/obj/item/reagent_container/glass/bottle/dexalin,
-		/obj/item/reagent_container/glass/bottle/inaprovaline,
-		/obj/item/reagent_container/glass/bottle/kelotane,
-		/obj/item/reagent_container/glass/bottle/oxycodone,
-		/obj/item/reagent_container/glass/bottle/peridaxon,
-		/obj/item/reagent_container/glass/bottle/tramadol,
-		)
-	var/list/stack_refill = list(
-		/obj/item/stack/medical/advanced/ointment,
-		/obj/item/stack/medical/advanced/bruise_pack,
-		/obj/item/stack/medical/ointment,
-		/obj/item/stack/medical/bruise_pack,
-		/obj/item/stack/medical/splint
-		)
+	var/list/chem_refill = null
+	var/list/stack_refill = null
```

Comment:
If these vars aren't being used anywhere else should be fine to remove them entirely.

Reply:
I want to see how it plays out first and if we decide after the test merge we want to keep it this way, I'll remove these variables and all references from then on.